gobblin-runtime/src/main/java/gobblin/runtime/job_catalog/CachingJobCatalog.java

```diff
@@ -61,6 +63,30 @@ public JobSpec getJobSpec(URI uri) throws JobSpecNotFoundException {
     }
   }
 
+  @Override
+  protected void startUp() {
+    _cache.startAsync();
```

Comment:
do you mean the fallback object? The cached object is a private field created by the `CachingJobCatalog`, so the `CachingJobCatalog` is the only object that can be responsible for starting/stopping it.